api/api/admin/__init__.py

```diff
@@ -83,3 +86,54 @@ class ProviderAdmin(admin.ModelAdmin):
     list_display = ("provider_name", "provider_identifier", "media_type")
     search_fields = ("provider_name", "provider_identifier")
     ordering = ("media_type", "provider_name")
+
+
+@admin.register(ThrottledApplication)
+class ThrottledApplicationAdmin(admin.ModelAdmin):
+    search_fields = ("client_id", "name")
+    list_display = ("client_id", "name", "user", "rate_limit_model")
+    ordering = ("client_id",)
+
+    def get_readonly_fields(self, request, obj=None):
```

Comment:
For both this and the class below - since we're not actually doing any modifications of the read only field list itself, we can use the [`readonly_fields` property](https://docs.djangoproject.com/en/5.0/ref/contrib/admin/#django.contrib.admin.ModelAdmin.readonly_fields) instead of having to define a `get_readonly_fields` function. Do you mind changing these to that pattern instead? Here's an example:

https://github.com/WordPress/openverse/blob/57223c279408fe2372503a052f5b179fc9446aa8/api/api/admin/__init__.py#L70